src/http/conn.rs

```diff
@@ -490,7 +497,7 @@ enum Reading {
 
 enum Writing<B> {
     Init,
-    Body(Encoder, Option<Cursor<B>>),
+    Body(Encoder, Option<Cursor<B>>, BytesMut),
```

Comment:
It depends on resources of your computer. I tend to stick it to 1 thread, cause I use a shared server, and a duration of 10 seconds. So `-t1 -d10`. Then, I bump up the connections until it starts to reduce the req/s significantly, and then use the same configuration in the feature branch, and then compare req/s.